astroquery/noirlab/__init__.py

```diff
@@ -65,7 +65,7 @@ class Conf(_config.ConfigNamespace):
     Configuration parameters for `astroquery.noirlab`.
     """
     server = _config.ConfigItem(
-        ['https://astroarchive.noao.edu',
+        ['https://astroarchive.noirlab.edu',
```

Comment:
Thanks for making these fixed. I'm afraid the current noirlab module is at the brink of removal as per https://github.com/astropy/astroquery/issues/2417 (same sentiment as this one: https://github.com/astropy/astroquery/issues/2316), not to keep knowingly broken things around.